frontend/src/components/admin/AdminDashboard.js

```diff
@@ -27,30 +56,104 @@ export class AdminDashboard extends Component {
-          </div>
-          <div className="tab__content">
-            <MinespaceUserManagement />
-          </div>
+          <Tabs
+            activeKey={this.state.activeTab}
+            defaultActiveKey="summary"
+            onChange={this.handleTabChange}
+            size="large"
+            animated={{ inkBar: true, tabPane: false }}
+          >
+            <TabPane tab="Verified Mines" key="verifiedMines">
+              <div className="tab__content">
+                <div>
+                  <h4>{this.state.verifiedMines.length}&nbsp;Verified Mines</h4>
+                  <div>
+                    {this.state.verifiedMines.length > 0 && (
+                      <AdminVerifiedMinesList
+                        minesVerifiedStatusList={this.state.verifiedMines.sort(
+                          this.compareMineName
+                        )}
+                      />
+                    )}
+                  </div>
+                </div>
+              </div>
+            </TabPane>
+            <TabPane tab="Mines to be Checked" key="unverifiedMines">
+              <div className="tab__content">
+                <div>
+                  <h4>{this.state.unverifiedMines.length}&nbsp;Mines Needing Re-Verification</h4>
+                  <div>
+                    {this.state.unverifiedMines.length > 0 && (
+                      <AdminVerifiedMinesList
+                        minesVerifiedStatusList={this.state.unverifiedMines.sort(
+                          this.compareMineName
+                        )}
+                      />
+                    )}
+                  </div>
+                </div>
+              </div>
+            </TabPane>
+            <TabPane tab="Pull Mine Manager History" key="mineManagerHistory">
+              <div className="tab__content">
+                <h2>Pull Mine Manager History</h2>
+                <Input
+                  placeholder="Enter a Mine Number"
+                  value={this.state.mineNo}
+                  onChange={this.handleMineManagerChange}
+                  style={{ width: "50%", "max-width": "300px" }}
+                />
+                <Button onClick={this.handleDownload}>Download History</Button>
+              </div>
+            </TabPane>
+            <TabPane tab="Manage Minespace Users" key="managerMinespaceUsers">
+              <div className="tab__content">
+                <MinespaceUserManagement />
+              </div>
+            </TabPane>
+          </Tabs>
         </div>
       </div>
     );
   }
 }
 
-export default AuthorizationGuard(Permission.ADMIN)(AdminDashboard);
+const mapDispatchToProps = (dispatch) =>
+  bindActionCreators(
+    {
+      fetchMineVerifiedStatuses,
+    },
+    dispatch
+  );
+
+AdminDashboard.propTypes = propTypes;
+
+export default compose(
+  connect(
+    null,
+    mapDispatchToProps
+  ),
+  AuthorizationGuard(Permission.ADMIN) // isPublic === true
```

Comment:
The `isPublic` comment isn't needed here, assuming this was copied over?

Reply:
fixed